pylib/cqlshlib/authproviderhandling.py

```diff
@@ -40,8 +40,7 @@ def _warn_for_plain_text_security(config_file, provider_settings):
         print("""\nNotice: Credentials in the cqlshrc file is deprecated and
         will be ignored in the future.\n
         Please use a credentials file to
-        specify the username and password.\n""",
-              file=sys.stderr)
+        specify the username and password.\n""")
```

Comment:
Same here, this shouldn't be changed, and it's also not related directly to the fix